docs/getting-started/downloading-export-batches-keys.md

```diff
@@ -11,6 +11,84 @@ server. Keys can be grouped by region and the frequency of batching is
 configurable. We recommend batches are configured to be generated at most every
 4 hours, with a likely time being somewhere between 4 and 24 hours.
 
+# Consuming Export Batches
+
+## Index File Structure
+
+The index.txt file indicates which export files are available for a mobile device
+to consume. The index file contains relative path names for available archives.
+A client is typically given two configuration items for pointing at a server
+
+* The absolute location of the index.txt file
+* The absolute base path to which the relative data in the index file should be appended
+
+As an example. If an index file is located at `https://cdn.example.com/24h/region/index.txt`
+and the export base path of `https://cdn.example.com/24h/` and the contents of the index
+file are:
+
+```
+region/file1.zip
+region/file21.zip
+region/other_export_.zip
+```
+
+Then the following 3 archives are are being offered to clients:
+
+1. `https://cdn.example.com/24h/region/file1.zip`
+1. `https://cdn.example.com/24h/region/file21.zip`
+1. `https://cdn.example.com/24h/region/other_export_.zip`
+
+And these are being arrived at by appending the contents of the index file (one per line)
+to the base path provided by the server operator.
+
+### Index Processing Guideliens
+
+* Do not make any assumptions or dependencies on the layout of the .zip filenames themselves.
+* Do not make any assumptions about the depth of the ralative path in the index file. 
+* The index.txt will always be ordered from oldest file to newest file.
+* Each file only needs to be processed once.
```

Comment:
\* Do not assume files will end with `.zip`, but _do_ assume that all files are in zip format.